docs/changelog.txt

```diff
@@ -75,6 +75,9 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Lua
 - ``overlay.reload()``: has been renamed to ``overlay.rescan()`` so as not to conflict with the global ``reload()`` function. If you are developing an overlay, please take note of the new function name for reloading your overlay during development.
 - ``gui``: changed frame naming scheme to ``FRAME_X`` rather than ``X_FRAME``, and added aliases for backwards compatibility. (for example ``BOLD_FRAME`` is now called ``FRAME_BOLD``)
+- ``gui.make_frame()``: now maps every tile sprite from a border spritesheet.
+- ``gui.paint_frame()``: Panels with width or height equal to 1 will now draw a ``divider``.
+- ``buildingplan``: has been updated to make use of the new gui elements internally.
```

Comment:
the `gui` functions are undocumented, and the `buildingplan` change is internal, so I think we can get by without a changelog entry for this PR

Reply:
yaaaaaay :D